docs/src/configuration/app/hooks/hooks-and-dependencies.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Use hooks with dependencies
+description: Manage dependencies for your hooks, such as compiling Sass files as part of your build.
+---
+
+If you use a specific package in a hook, you may want to manage dependencies for it.
+For example, you may want to compile Sass files as part of your build process.
```

Comment:
I guess a one-liner to explain what sass (and less) are, what it's used for, ... is not the scope of the page?

Reply:
I would try to avoid it. I'll try to add something to the style guide later to clarify this.